README.md

```diff
@@ -28,7 +28,18 @@ The API reference is located at [realm.io/docs/java/api](http://realm.io/docs/ja
 
 ## Using Snapshots
 
-If you want to test recent bugfixes or features that have not been packaged in an official release yet, you can use a **-SNAPSHOT** release of the current development version of Realm via Gradle, available on [JFrog](http://oss.jfrog.org/oss-snapshot-local/io/realm/realm-android/)
+If you want to test recent bugfixes or features that have not been packaged in an official release yet, you can use a **-SNAPSHOT** release of the current development version of Realm via Gradle, available on [OJO](http://oss.jfrog.org/oss-snapshot-local/io/realm/realm-android/)
+
+    buildscript {
+        repositories {
+            maven {
+                url 'http://oss.jfrog.org/artifactory/oss-snapshot-local'
+            }
+        }
+        dependencies {
+            classpath "io.realm:gradle-plugin:<version>-SNAPSHOT"
```

Comment:
Yeah, but considering the extend we go to only have to update version once (actually, not at all when this is merged), i really dislike the idea of having to manually edit this file on every release.
And unfortunately I don't see a way to do this with the release plugin without forking it or contributing to it and having to wait for its next release